frontend/src/pages/QADashboard/Components/RecipientCard.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from '@trussworks/react-uswds';
+// import { TRAINING_REPORT_STATUSES } from '@ttahub/common';
+import { v4 as uuidv4 } from 'uuid';
+import { Link } from 'react-router-dom';
+import { checkForDate } from '../../../utils';
+import ExpanderButton from '../../../components/ExpanderButton';
+import GoalCard, { goalPropTypes } from './GoalCard';
+import { getScoreBadge } from '../../../components/ClassScoreBadge';
+
+import './RecipientCard.scss';
+
+function RecipientCard({
+  recipient,
+  handleGoalCheckboxSelect,
+  isChecked,
+  zIndex,
+}) {
+  const [goalsExpanded, setGoalsExpanded] = useState(false);
+  const closeOrOpenGoals = () => {
+    setGoalsExpanded(!goalsExpanded);
+  };
+  return (
+    <>
+      <article
+        className="ttahub-recipient-card usa-card padding-3 radius-lg border width-full maxw-full smart-hub-border-base-lighter position-relative"
+        data-testid="recipientCard"
+        style={{ zIndex }}
+      >
+        <div className="ttahub-recipient-card__row position-relative">
+          <div className="ttahub-recipient-card__recipient-column__cb margin-top-1">
+            <Checkbox
+              id={`recipient-select-${recipient.name}`}
+              label=""
+              value={recipient.id}
+              checked={isChecked}
+              onChange={handleGoalCheckboxSelect}
+              aria-label={`Select recipient ${recipient.name}`}
+              className="margin-right-1"
+              data-testid="selectRecipientTestId"
+            />
+          </div>
+          <div className="ttahub-recipient-card__recipient-column__break" />
```

Comment:
why do we need an empty element?

Reply:
This fixes an issue when the window is shrunk shown below (let me know if you have time to go over a better solution):

Good
![image](https://github.com/user-attachments/assets/064fe0c4-bdee-434e-b896-02373741f941)

Bad
![image](https://github.com/user-attachments/assets/55d77909-91d8-4979-a842-f5bdc0be89ab)

Reply:
Neverminded this has been removed.